copier/main.py

```diff
@@ -326,6 +330,10 @@ def answers(self) -> AnswersMap:
             if var_name in result.init:
                 # Do not ask again
                 continue
+
+            if self.default_overrides.get(var_name) is not None:
+                details['default'] = self.default_overrides[var_name]
```

Comment:
Just in case you want a specific `None` default, I think it's better to:

```suggestion
            with supress(KeyError):
                details['default'] = self.default_overrides[var_name]
```